src/xmipp/applications/scripts/tomogram_reconstruction/tomogram_reconstruction.py

```diff
@@ -130,68 +135,80 @@ def defineParams(self):
+        self.addExampleLine('xmipp_tomogram_reconstruction --tiltseries ts.mrc --angles angles.xmd --thickness 300 --gpu 0 -o tomogram.mrc --method os-asd-pocs')
+        self.addExampleLine('xmipp_tomogram_reconstruction --tiltseries ts.mrc --angles angles.xmd --thickness 300 --gpu 0 -o tomogram.mrc --method awasd-pocs')
+        self.addExampleLine('xmipp_tomogram_reconstruction --tiltseries ts.mrc --angles angles.xmd --thickness 300 --gpu 0 -o tomogram.mrc --method irn-tv-cgls')
+        self.addExampleLine('xmipp_tomogram_reconstruction --tiltseries ts.mrc --angles angles.xmd --thickness 300 --gpu 0 -o tomogram.mrc --method ab-gmres')
+        self.addExampleLine('xmipp_tomogram_reconstruction --tiltseries ts.mrc --angles angles.xmd --thickness 300 --gpu 0 -o tomogram.mrc --method ab-gmres')
+        self.addExampleLine('xmipp_tomogram_reconstruction --tiltseries ts.mrc --angles angles.xmd --thickness 300 --gpu 0 -o tomogram.mrc --method hybrid_lsqr')
+        self.addExampleLine('xmipp_tomogram_reconstruction --tiltseries ts.mrc --angles angles.xmd --thickness 300 --gpu 0 -o tomogram.mrc --method lsqr')
+        self.addExampleLine('xmipp_tomogram_reconstruction --tiltseries ts.mrc --angles angles.xmd --thickness 300 --gpu 0 -o tomogram.mrc --method cgls')
         
 
     def readInputParams(self):
         '''
         In this function the parameters are read. For for information about their use see the help
         '''
-        
+
         self.fnTs          = self.getParam('--tiltseries')
         self.fnAngles      = self.getParam('--angles')
         self.thickness     = self.getIntParam('--thickness')
         self.useAstra      = self.checkParam('--astra')
-        self.method        = self.getParam('--method').lower()
+        self.method        = self.fixMethod(self.getParam('--method').lower())
         self.fnOut         = self.getParam('-o')
+        self.normalizeTi   = self.getParam('--normalize').lower()
         self.gpuId         = self.getIntParam('--gpu') if self.checkParam('--gpu') else 0
         
         if self.method == 'wbp':
             self.method = 'fbp'
         
         if self.method == 'fdk' or self.method == 'fbp':
-            self.filterToApply = self.getParam('--filter') if self.checkParam('--filter') else 'ram_lak'
-            self.checkFilter(self.filterToApply)
+            self.filterToApply = self.getParam('--filter').lower() if self.checkParam('--filter') else 'ram_lak'
+            self.checkFilter(self.filterToApply, self.method)
             
         if self.method == 'sirt' or self.method == 'sart' or self.method == 'ossart':
             self.iterations    = self.getIntParam('--iter') if self.checkParam('--iter') else 20
-            self.lmbda         = self.getDoubleParam('--lmbda') if self.checkParam('--lmbda') else 1.0
+            self.lmbda         = self.getDoubleParam('--lambda') if self.checkParam('--lambda') else 1.0
```

Comment:
Other variables that have something to do with lambda have the complete name "lambda". Maybe change self.lmbda to self.lambda for uniformity?

Reply:
I did it at the beginning, but lambda is a python words, and that created some issues. So, I put lmbda :)